README.md

```diff
@@ -134,61 +188,84 @@ To test quipucords against virtual machines running on a cloud provider, view th
+folder, like shown below:  
+/quipucords  
+   --quipucords  
+   --quipucords-ui
+
+See [quipucords-ui installation instructions](https://github.com/quipucords/quipucords-ui) for information.  
+You will need to have NodeJS installed. See [Nodejs](<https://nodejs.org/>) official website for instructions.  
+   On Mac:
+   ```
+   brew install yarn (if you don't already have yarn)
+   make build-ui
+   ```
+ 
+   On Linux:
+   
+   ```
+   npm install yarn (if you don't already have yarn)
+   make build-ui
+   ```
+3. Build the Docker image through Docker-compose:
+
+   For Linux users using Podman instead of Docker, this one-time setup is necessary:
+   ```
+   systemctl enable --user podman.socket
+   systemctl start --user podman.socket
+   # add the next line to your ~/.bashrc (or equivalent)
+   export DOCKER_HOST=unix://$XDG_RUNTIME_DIR/podman/podman.sock
+   ```
+   then run:
+   ```
+   docker-compose up -d
+   ```
+   _NOTE:_ The need to use ``sudo`` for this step is dependent upon on your system configuration.  
+
+   For Mac users:
+   ```
+   docker-compose up -d
+   ```
 4. There are many different options for running the QPC server.
 
    A. Run the Docker image with Postgres container:
     ```
-    docker run --name qpc-db -e POSTGRES_PASSWORD=password -d postgres:14.1
-    export QPC_VAR_DATA=$PWD/var/data
-    mkdir -p $QPC_VAR_DATA
-    docker run --name quipucords --link qpc-db:qpc-link -d -e QPC_DBMS_HOST=qpc-db -p 9443:443 -v $QPC_VAR_DATA:/var/data -i quipucords:0.0.46
+    docker-compose up -d
     ```
 
    B. Run the Docker image with external Postgres container:
     ```
     ifconfig (get your computer's external IP if Postgres is local)
-    docker run -d --name quipucords -e "QPC_DBMS_PASSWORD=password" -e"QPC_DBMS_HOST=EXTERNAL_IP" -p 9443:443 -i quipucords:0.0.46
+    docker-compose up -d --name quipucords -e "QPC_DBMS_PASSWORD=password" -e"QPC_DBMS_HOST=EXTERNAL_IP" -p 9443:443 -i quipucords:0.0.46
     ```
 
    C. Run the Docker image with SQLite:
     ```
-    docker run -d --name quipucords -e "QPC_DBMS=sqlite" -p 9443:443 -i quipucords:0.0.46
+    docker-compose up -d --name quipucords -e "QPC_DBMS=sqlite" -p 9443:443 -i quipucords:0.0.46
     ```
 
    D. For debugging purposes you may want to run the Docker image with the `/app` directory mapped to your local clone of quipucords and the logs mapped to a temporary directory. Mapping the `/app` directory allows you to rapidly change server code without having to rebuild the container. Mapping the logs to `/tmp` allows you to tail a local copy without having to exec into the container.
     ```
-    docker run -d --name quipucords -e "QPC_DBMS=sqlite" -p 9443:443 -v /path/to/local/quipucords/:/app -v /tmp:/var/log -i quipucords:0.0.46
+    docker-compose up -d --name quipucords -e "QPC_DBMS=sqlite" -p 9443:443 -v /path/to/local/quipucords/:/app -v /tmp:/var/log -i quipucords:0.0.46
```

Comment:
```suggestion
    docker run -d --name quipucords -e "QPC_DBMS=sqlite" -p 9443:443 -v /path/to/local/quipucords/:/app -v /tmp:/var/log -i quipucords:0.0.46
```

Reply:
Alternatively we could also drop "D". `/app` is mounted on "A" (through compose config) and since we dropped supervisor the logs are available when running `docker logs <container name>` or `docker-compose logs qpc-server`.

What do you think, @gtanzillo ?